src/invoices/Voucher/VoucherViewLayer.js

```diff
@@ -0,0 +1,125 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import { FormattedMessage } from 'react-intl';
+import { get } from 'lodash';
+
+import {
+  Button,
+  Col,
+  Icon,
+  Layer,
+  MenuSection,
+  Pane,
+  PaneMenu,
+  Row,
+} from '@folio/stripes/components';
+import {
+  AppIcon,
+  stripesConnect,
+} from '@folio/stripes/core';
+
+import { LoadingPane } from '../../common/components';
+import {
+  invoiceResource,
+  VOUCHER_BY_ID,
+  VOUCHER_LINES,
+} from '../../common/resources';
+import VoucherView from './VoucherView';
+
+const VoucherViewLayer = ({ match: { params }, mutator, resources }) => {
+  const voucher = get(resources, ['voucher', 'records', 0]);
+  const voucherLines = get(resources, ['voucherLines', 'records'], []);
+  const vendorInvoiceNo = get(resources, ['invoice', 'records', 0, 'vendorInvoiceNo']);
+  const isLoading = !get(resources, ['voucher', 'hasLoaded']);
+
+  const closeVoucher = useCallback(
+    () => {
+      const _path = `/invoice/view/${params.id}`;
+
+      mutator.query.update({ _path });
+    },
+    [params.id],
+  );
+
+  const renderActionMenu = () => (
+    <MenuSection id="voucher-actions">
+      <Button
+        buttonStyle="dropdownItem"
+        data-test-edit-voucher-button
+      >
+        <Icon size="small" icon="edit">
+          <FormattedMessage id="ui-invoice.button.edit" />
+        </Icon>
+      </Button>
+    </MenuSection>
+  );
+
+  const lastMenu = (
+    <PaneMenu>
+      <Button
+        marginBottom0
+        buttonStyle="primary"
+      >
+        <FormattedMessage id="ui-invoice.button.edit" />
+      </Button>
+    </PaneMenu>
+  );
+
+  if (isLoading) {
+    return (
+      <Layer isOpen>
+        <LoadingPane onClose={closeVoucher} />
+      </Layer>
+    );
+  }
+
+  return (
+    <Layer isOpen>
+      <Pane
+        actionMenu={renderActionMenu}
+        appIcon={<AppIcon app="invoice" size="small" />}
+        defaultWidth="fill"
+        dismissible
+        id="pane-voucher"
+        lastMenu={lastMenu}
+        onClose={closeVoucher}
+        paneSub={<FormattedMessage id="ui-invoice.voucher.paneSubTitle" values={{ number: voucher.voucherNumber }} />}
```

Comment:
IMHO, you should break such long lines.
```
paneSub={(
  <FormattedMessage 
    id="ui-invoice.voucher.paneSubTitle" 
    values={{ number: voucher.voucherNumber }}
  />
)}
```

Reply:
Also, `values={{ voucherNumber: voucher.voucherNumber }}` looks more specific

Reply:
Fixed